octoprint_octorelay/test_driver.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+import unittest
+import sys
+from unittest.mock import Mock
+
+GPIO_mock = sys.modules["RPi.GPIO"] # mocked in test__init.py
```

Comment:
in both test files GPIO mocking should be make conditional, so each file could be ran individually as well